packages/ListBoxWithTags/src/ListBoxWithTags.js

```diff
@@ -74,6 +74,8 @@ const renderTrigger = ({ t, size, selectedOptions, onRemove, renderTag, tagLabel
 
   return (
     <sc.Trigger
```

Comment:
The default label for this trigger button gets unnecessarily long and redundant because it includes the desciption of all of the remove buttons too.  I thin we should mitigate that by providing an `aria-label` (`a11yText`) that can be composed more thoughtfully.